plot.py

```diff
@@ -663,16 +668,19 @@ def _set_default_limits(self):
                 
     def _check_file_type(self):
```

Comment:
Good idea. I actually ended up pulling the limits initialization into this with the last PR (#36).